app/src/main/kotlin/com/wire/android/ui/common/topappbar/CommonTopAppBarViewModel.kt

```diff
@@ -77,6 +78,16 @@ class CommonTopAppBarViewModel @Inject constructor(
         }
     }
 
+    private fun legalHoldStatusFlow(userId: UserId) = coreLogic.sessionScope(userId) {
+        observeLegalHoldRequest() // TODO combine with legal host status
+            .map { legalHoldRequestResult ->
+                when (legalHoldRequestResult) {
+                    is ObserveLegalHoldRequestUseCaseResult.ObserveLegalHoldRequestAvailable -> LegalHoldStatus.PENDING
```

Comment:
We may need to rename `ObserveLegalHoldRequestAvailable` to just `LegalHoldRequestAvailable` in kalium

Reply:
Yeah, but it's just for better readability, so I won't raise a kalium PR just for that and will try to remember to fix that in the next kalium PR